src/Service/MigrationService.php

```diff
@@ -170,33 +155,12 @@ public function getMigrationHistory(?int $limit = null): array
                 continue;
             }
 
-            if (preg_match('/m?(\d{6}_?\d{6})(\D.*)?$/is', $row['version'], $matches)) {
```

Comment:
Is that not needed anymore?

Reply:
Not need. Now use `id` for correct order.